src/lib/packet.h

```diff
@@ -565,21 +572,44 @@ typedef enum {
 
 #define PGP_SA_DEFAULT_CIPHER PGP_SA_CAST5
 
+#define PGP_SA_DEFAULT_BLOCK_CIPHER_MODE PGP_BLOCK_CIPHER_MODE_CFB
+
 void pgp_calc_mdc_hash(
   const uint8_t *, const size_t, const uint8_t *, const unsigned, uint8_t *);
 unsigned pgp_is_hash_alg_supported(const pgp_hash_alg_t *);
 
+/* Maximum block size for symmetric crypto */
+#define PGP_MAX_BLOCK_SIZE 16
+
+/* Maximum key size for symmetric crypto */
+#define PGP_MAX_KEY_SIZE 32
+
+/* Salt size for hashing */
+#define PGP_SALT_SIZE 8
+
+#define PGP_NONCE_SIZE 12
```

Comment:
Probably best to move this to `key_store_g10.c` and name it something like `G10_OCB_NONCE_SIZE` (or even `G10_OCB_IV_SIZE`) since it's not part of the RFC.